doc/deployment/amazon_web_services.md

```diff
@@ -223,10 +225,11 @@ pachd               1.6.0
 
 ## One Shot Script
 
-### Install additional prerequisites
-
-This scripted deploy requires a couple of prerequisites in addition to the ones listed under [Prerequisites](amazon_web_services.md#prerequisites):
+### Prerequisites
 
+- [AWS CLI](https://aws.amazon.com/cli/) - have it installed and have your [AWS credentials](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html) configured.
```

Comment:
We should add pachctl as a prereq. Users often forget that.

Reply:
It's included explicitly here as part of the Pachyderm installation: https://github.com/pachyderm/pachyderm/blob/b593fc2c567d9dc4310e6cf6ea495ebf1b430db8/doc/deployment/amazon_web_services.md#install-pachctl. But I'm fine with moving it anywhere. If we move it up to the top, we should do that in all of the deploy docs and then maybe just have a common doc somewhere that talks about installing pachctl.